app/Models/Post.php

```diff
@@ -177,6 +177,44 @@ public function toBlinkPayload()
         ];
     }
 
+    /**
+     * Transform the post model for Quasar.
+     *
+     * @return array
+     */
+    public function toQuasarPayload()
+    {
+        return [
+            'id' => $this->id,
+            'signup_id' => $this->signup_id,
+            'northstar_id' => $this->northstar_id,
+            'type' => $this->type,
+            'action' => $this->action,
+            'quantity' => $this->quantity,
+            // Add cache-busting query string to urls to make sure we get the
+            // most recent version of the image.
+            // @NOTE - Remove if we get rid of rotation.
+            'media' => [
```

Comment:
have we always been giving them these two urls? I feel like we should only need to sent them the `original_url` since they don't do any work displaying the images and shouldn't care about edited vs not edited versions.

Reply:
@sbsmith86 yeah, this is what the activity endpoint returns for a post too.

@sheyd do you use both of these values? `url` and `original_image_url`?

Reply:
@katiecrane I currently only ingest the ['media']['url'] part, so I'm guessing that's the `url`. The reason we wanted the URI on Quasar side was for the Campaigns team to be able to directly click on a reportback from Looker and see the media asset. Whatever URI enables this would be preferred.

Reply:
@sheyd either could do that, but the one you currently use is the re-sized square crop one that we display elsewhere so let's stick with that one unless the admins would need the fullsize original image?

Reply:
Re-sized square crop should be 💯